modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/actionItems/handleActionClick.ts

```diff
@@ -62,7 +66,9 @@ const handleActionClick = ({
 
 	const doAction = ({defaultPrevented}: {defaultPrevented: boolean}) => {
 		if (target === INFO_PANEL && onInfoPanelToggleButtonClick) {
-			onInfoPanelToggleButtonClick();
+			selectItems?.({trigger: 'body', value: itemId});
```

Comment:
We should not be pushing down `selectItems` function, that belongs in root component. A better architecture is  to have generic `onClick` or `onSelect` event that propagates up to root component, with a callback defined there.